bin/oneoff/backfill_data/user_school_infos_backfill

```diff
@@ -6,21 +6,22 @@ num_users_updated = 0
 batch_size = 10000
 
 puts "Processing..."
-
+start = Time.now
 User.where.not(school_info: nil).where(user_type: "teacher").find_in_batches(batch_size: batch_size) do |batch|
   ActiveRecord::Base.transaction do
     batch.each do |user|
-      UserSchoolInfo.create!(
-        user_id: user.id,
-        school_info_id: user.school_info_id,
+      user.user_school_infos.where(school_info_id: user.school_info_id).first_or_create(
         start_date: user.created_at,
-        end_date: nil,
         last_confirmation_date: user.created_at
```

Comment:
Looking back at Poorva's spec, here's what she had for `last_confirmation_date`:
last_confirmation_date should be when the user filled out the interstitial last (last_seen_school_info_interstitial). If they’ve never seen the interstitial, then it should be their account creation date. 

So thinking there should be some conditional logic that first looks to see if the user has a value for `last_seen_school_info_interstitial`, which is in their `properties`

Reply:
@bencodeorg Code updated